src/AddLot.js

```diff
@@ -2,25 +2,44 @@ import React from 'react';
 import * as Utilities from './utilities';
 
 class AddLot extends React.Component {
-  state = { value: null, submitted: false };
+  state = { tickerValue: null, boughtShares: null, submitted: false };
 
-  handleChange = (event) => {
-    this.setState({ value: event.target.value });
+  handleTickerChange = (event) => {
+    this.setState({ tickerValue: event.target.value });
+  };
+
+  handleSharesChange = (event) => {
+    this.setState({ boughtShares: event.target.value });
   };
 
   handleSubmit = (event) => {
     event.preventDefault();
     this.props.submitted(true);
-    Utilities.addLotToPortfolio(this.state.value);
+    Utilities.addLotToPortfolio(
```

Comment:
Curious why you have this very spread-out style. I prefer things a bit tighter, e.g.

`Utilities.addLotToPortfolio(this.state.tickerValue, this.state.boughtShares);`

all on one line. This allows taking in a larger chunk of code when reading it.

Reply:
just auto-formatting :) line length.